### DISCUSSION TITLE
How to deal with large packages.(chunk load error)

### DISCUSSION THREAD

**Discussion by awahab3109** [vercel/next.js#65662] [App Router] [answered]
So basically I am using matrix-js-sdk in my project. Now the problem comes with this package is it implicitly add this file ("matrix_sdk_crypto_wasm_bg.wsm) . I used next bundle analyzer and it shows it's size as 2.8 mb (gzipped). Now next js very rarely (I don't want that behavoir rarely too) throws a timeout chunk load error. because file is 2.8 mb and it take time to download which causes timeout chunk load error. what should I do in this case . Can someone guide me.
![Screenshot from 2024-05-12 23-10-13](https://github.com/vercel/next.js/assets/122437993/1b2da924-5846-4a95-ae3b-28402b3bd874)

**Comment by SouthLink** [comment, marked as answer]
If your package is too large and you want to avoid loading failures or affecting page performance, you may need to [load dynamically](https://nextjs.org/docs/pages/building-your-application/optimizing/lazy-loading#nextdynamic)
